README.md

```diff
@@ -11,81 +13,36 @@ Client library for accessing Gro Intelligence's agricultural data platform.
-
-### Install with git clone 
-
-```
-git clone https://github.com/gro-intelligence/api-client.git  [<your-gro-path>]
+pip install git+https://github.com/gro-intelligence/api-client.git
 ```
-If you don't specify your gro installation path, it will be the current directory.
-In addition, optionally you can:
-
-1. [Set `PYTHONPATH=<your-gro-path>` as an environment variable](https://www.techwalla.com/articles/how-to-set-your-python-path)
-
-2. Make `gro_client` an alias for `python <your-gro-path>/gro_client.py`
-
 
 ## Gro API authentication token
 
-### Getting a token
+Use the Gro web application to retrieve an authentication token (instructions are in the wiki [here](https://github.com/gro-intelligence/api-client/wiki/Authentication-Tokens#11-using-the-gro-web-application-preferred)).
 
-You can use the gro_client command line tool to request an authentication token. Note that you will be prompted to enter a password.
+### Saving your token as an environment variable (optional)
 
-```sh
-gro_client --user_email=email@example.com --print_token
-```
-
-This token is used throughout the Gro API client code, wherever authentication is required.
-
-Gro authentication tokens *may* expire or be reset by the administrator,
-depending on your account status. You can re-run the above command to get a new one 
-if your previous one has expired.
-
-### Saving your token as an environment variable
-
-If you don't want to enter a password or token each time, you can save
-the token as an environment variable. Please consult your OS or IDE documentation for information on how to set environment variables, e.g. [setting environment variables in Windows Powershell](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-6) and [Mac OS X/Linux](https://apple.stackexchange.com/questions/106778/how-do-i-set-environment-variables-on-os-x).  In some of the sample code, it is assumed that you have the token saved to your environment variables as GROAPI_TOKEN. 
+If you don't want to enter a password or token each time, you can save the token as an environment variable. Please consult your OS or IDE documentation for information on how to set environment variables, e.g. [setting environment variables in Windows Powershell](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-6) and [Mac OS X/Linux](https://apple.stackexchange.com/questions/106778/how-do-i-set-environment-variables-on-os-x). In some of the sample code, it is assumed that you have the token saved to your environment variables as `GROAPI_TOKEN`.
 
 ## Examples
 
 Navigate to [api/client/samples/](api/client/samples/) folder and try executing the provided examples.
 
-Try [quick_start.py](api/client/samples/quick_start.py)
+1. Try [quick_start.py](api/client/samples/quick_start.py).
```

Comment:
I think this would be a good opportunity to add context around these samples and why users might find it helpful. E.g. "This quick start file demonstrations some helpful frameworks for engaging with the Gro API Client. The indicates the entities of interest, uses the get_data_series function to return all related series, sets a time range restriction, then retrieves get_data_points for all series, and finally outputs the request into a CSV"

Reply:
Added. I want to put more detailed info in the quick_start file itself as in https://github.com/gro-intelligence/api-client/pull/59/files#diff-39a3741db0aea5fdfb9fdd7bb405c162R1 so don't want to duplicate too much, but I added some of the most important info on what to expect from it